# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/CacheConstants.h
# Commit: 388a15605114
# Full Hash: 388a156051146d9c128adc05365f9c1e0aa94cb2
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-05-27 15:58:57
# Regressor Bug: 1737919
# File Overlap Count: 1
# Description:
#   Bug 1737919 part 3: Support spelling errors for cached RemoteAccessibles. r=morgan
#   
#   Even though spelling errors can cross Accessibles and are represented in the DOM as ranges, we cache them for each text leaf.
#   This is necessary so that we correctly update the offsets when text is inserted or removed from a leaf.
#   We cache them as an array of offsets, including both the start and end offset for each range.
# ==============================================================================

diff -r 08d033b9651b -r 388a15605114 accessible/base/CacheConstants.h
--- a/accessible/base/CacheConstants.h	Fri May 27 10:56:40 2022 +0000
+++ b/accessible/base/CacheConstants.h	Fri May 27 10:56:41 2022 +0000
@@ -25,6 +25,7 @@
   static constexpr uint64_t TransformMatrix = ((uint64_t)0x1) << 10;
   static constexpr uint64_t ScrollPosition = ((uint64_t)0x1) << 11;
   static constexpr uint64_t Table = ((uint64_t)0x1) << 12;
+  static constexpr uint64_t Spelling = ((uint64_t)0x1) << 13;
   static constexpr uint64_t All = ~((uint64_t)0x0);
 };
 